### ISSUE TITLE
SPECT logic

### ISSUE THREAD

**Issue by gtm2122** [guanhuaw/MIRTorch#32]
`for y in range(image.shape[1]):
        mumapr = 0.5 * rot_mumap[:, y, :]  # initialize
        sum_mumapr = mumapr + torch.sum(rot_mumap[:, 0:y, :], dim=1)  # add 0,1,...,y-1
        exp_mumapr = torch.exp(sum_mumapr * -dy)
        mu_rot_image = (
            rot_image[:, y, :] * exp_mumapr
        )  # apply depth-dependent attenuation
        view_list[:, y, :] = fft_conv(mu_rot_image, psf[:, :, y])
    return torch.sum(view_list, dim=1)  # nx * nz`

In this setting is it assumed that the detector is a flat detector with 0 degree acceptance angle ?

After rotating the image is the detector on the left side of the rotated image ?

Why is the summation depth dependent, for a given row is this trying to sum all the attenuated activity along the path starting from each source to detector ? Can you provide some example of the activity map and attenuation map ?
